bench-vortex/src/engines/ddb/mod.rs

```diff
@@ -69,38 +63,28 @@ pub fn get_executable_path(user_supplied_path_flag: &Option<PathBuf>) -> PathBuf
         }
     }
 
-    let duckdb_vortex_path = PathBuf::from_str(&repo_root.unwrap_or_else(|| ".".to_string()))
+    PathBuf::from_str(&repo_root.unwrap_or_else(|| ".".to_string()))
         .expect("failed to find the vortex repo")
-        .join("duckdb-vortex");
-
-    let duckdb_path = duckdb_vortex_path.join("build/release/duckdb");
+        .join("duckdb-vortex")
+}
 
-    validate_path(&duckdb_path);
+pub fn vortex_duckdb_extension_path() -> PathBuf {
+    vortex_duckdb_folder().join("build/release/extension/vortex/vortex.duckdb_extension")
+}
 
-    duckdb_path
+pub fn get_executable_path(user_supplied_path_flag: &Option<PathBuf>) -> PathBuf {
```

Comment:
Not necessarily in this PR but we should rename `get_executable_path` to something like `duckdb_executable_path`.